src/tlo/cli.py

```diff
@@ -234,6 +234,25 @@ def batch_submit(ctx, scenario_file, asserts_on, more_memory, keep_pool_alive, i
     task_dir = "${{AZ_BATCH_TASK_DIR}}"
     gzip_pattern_match = "{{txt,log}}"
     command = f"""
+    # Logfile for `earlyoom`
+    EARLYOOM_LOG=/tmp/earlyoom.log
+
+    function terminate_and_log() {{{{
+        # Terminate the `earlyoom` process
+        kill "${{{{EARLYOOM_PID}}}}"
+        # Print line(s) in the log file referencing terminated processes, if any.
+        grep "^sending SIGTERM to process" "${{{{EARLYOOM_LOG}}}}"
```

Comment:
This catches also SIGKILL:
```suggestion
        grep "^sending \w+ to process" "${{{{EARLYOOM_LOG}}}}"
```